python/lsst/analysis/drp/plotUtils.py

```diff
@@ -295,7 +309,56 @@ def perpDistance(p1, p2, points):
     dists = []
     for point in points:
         point = np.array(point)
-        distToLine = np.cross(p1 - point, p2 - point)/np.linalg.norm(p2 - point)
+        distToLine = np.cross(p1 - point, p2 - point)/np.linalg.norm(p1 - point)
         dists.append(distToLine)
 
     return dists
+
+
+def mkColormap(colorNames):
+    """Make a colormap from the list of color names.
+
+    Parameters
+    ----------
+    colorNames : `list`
+        A list of strings that correspond to matplotlib
+        named colors.
+
+    Returns
+    -------
+    cmap : `matplotlib.colors.LinearSegmentedColormap`
+    """
+
+    nums = np.linspace(0, 1, len(colorNames))
+    blues = []
+    greens = []
+    reds = []
+    for (num, color) in zip(nums, colorNames):
+        r, g, b = colors.colorConverter.to_rgb(color)
+        blues.append((num, b, b))
+        greens.append((num, g, g))
+        reds.append((num, r, r))
+
+    colorDict = {"blue": blues, "red": reds, "green": greens}
+    cmap = colors.LinearSegmentedColormap("newCmap", colorDict)
+    return cmap
+
+
+def extremaSort(xs):
+    """Return the ids of the points reordered so that those
+    furthest from the median, in absolute terms, are last.
+
+    Parameters
+    ----------
+    xs : `np.array`
+        An array of the values to sort
+
+    Returns
+    -------
+    ids : `np.array`
+    """
+
+    med = np.median(xs)
+    dists = np.fabs(xs - med)
```

Comment:
There's essentially no good reason to use `np.fabs` over `np.abs`, as far as I can tell, and `np.abs` is more readable.